server/src/test/java/keywhiz/auth/ldap/LdapAuthenticatorTest.java

```diff
@@ -101,24 +117,7 @@ public void ldapAuthenticatorRejectsInvalidUsername() throws Exception {
     assertThat(missingUser.isPresent()).isFalse();
   }
 
-  private static class IsDnSearch extends ArgumentMatcher<SearchRequest> {
-    @Override
-    public boolean matches(Object o) {
-      if (o == null) return false;
-      SearchRequest request = (SearchRequest) o;
-      return request.getBaseDN().equals("ou=users,dc=example,dc=com");
-    }
-  }
-
-  private static class IsRoleSearch extends ArgumentMatcher<SearchRequest> {
-    @Override
-    public boolean matches(Object o) {
-      if (o == null) return false;
-      SearchRequest request = (SearchRequest) o;
-      return request.getBaseDN().equals("ou=roles,dc=example,dc=com");
-    }
-  }
-
+  @Ignore
```

Comment:
Justify the `@Ignore` here

Reply:
Can we just add a comment for now, and we can fix the tests later?

Reply:
(resolved offline, @mcpherrinm is to add the comments)

Reply:
I'm doing this